src/components/viewarticle_components/CommentCard.jsx

```diff
@@ -16,6 +21,19 @@ const CommentCard = ({ comment }) => {
     })
   }, [])
 
+  const deleteOnClickHandler = () => {
+    setProcessingDeleteRequest(true);
+    deleteCommentById(comment.comment_id).then((response) => {
+      if (response.name === "AxiosError") {
+        setFeedback(`Error: ${response.message}`);
+        setProcessingDeleteRequest(false);
```

Comment:
If you move the catch from the api.js file into this handler and chain it on deleteCommentById you won't have to check the response.name, you can just deal with it in the catch block

Reply:
Great idea. Thanks!